use_cases/ecs-ec2-daemon-collector/task-definition-templates/cloudwatch-task-logs.json

```diff
@@ -0,0 +1,83 @@
+{
+    "family": "fluent-bit-daemon-collector-cloudwatch",
+    "networkMode": "host",
+    "taskRoleArn": "{INSERT YOUR TASK ROLE ARN}", 
+    "executionRoleArn": "{INSERT YOUR EXECUTION ROLE ARN}",
+    "volumes": [
+        {
+            "name": "applogs",
+            "host": {
+                "sourcePath": "/var/lib/docker/containers"
+            }
+        },
+        {
+            "name": "fluent-bit-buffer-directory",
+            "host": {
+                "sourcePath": "/var/fluent-bit"
+            }
+        }
+    ],
+    "containerDefinitions": [
+        {
+            "essential": true,
+            "image": "public.ecr.aws/aws-observability/aws-for-fluent-bit:init-stable",
+            "name": "fluent-bit-daemon-collector",
+            "logConfiguration": {
+                "logDriver": "awslogs",
+                "options": {
+                    "awslogs-group": "/aws/ecs/{INSERT CLUSTER_NAME}/collector",
+                    "awslogs-region": "{INSERT REGION}",
+                    "awslogs-create-group": "true",
+                    "awslogs-stream-prefix": "fluent-bit-stdout-stderr-"
+                }
+            },
+            "environment": [
```

Comment:
Need to add an env var here for IMDS version used by AWS Filter.

Reply:
Actually, since we use host mode, its always safe to use V2 with default hop limit of one, so will just use that. No need to worry about this. V2 is always enabled by default now.